doc/release-notes.rst

```diff
@@ -90,34 +93,47 @@ Major Changes from Kraken
     ``osd_snap_trim_sleep``, and ``osd_scrub_sleep`` have been
     reimplemented to work efficiently.  (These are used in some cases
     to work around issues throttling background work.)
+  * The crush ``choose_args`` encoding has been changed to make it
+    architecture-independent.
```

Comment:
this compat note should go in the upgrade notes down below.

Reply:
ok